test/core/core_online_test.cpp

```diff
@@ -154,20 +153,20 @@ void TestCoreOnline::change_name()
 
     alice->setUsername(aliceName);
 
-    QTRY_VERIFY_WITH_TIMEOUT(aliceSaveRequest.count() == 1, local_timeout);
```

Comment:
I'm guessing the local_timeout ones weren't actually timing out and you just normalized the timeouts a bit as well?